lib/yast/rake.rb

```diff
@@ -43,9 +26,9 @@ def self.spec_version
 
 task.enhance(prerequisites)
 
+Yast::Tasks.submit_to(:factory)
```

Comment:
Please make it also ENV configurable, so we can use similar trick in jenkins for internal instance.